packages/contracts-bedrock/src/L2/WETH.sol

```diff
@@ -0,0 +1,21 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import { WETH98 } from "src/dispute/weth/WETH98.sol";
+import { Predeploys } from "src/libraries/Predeploys.sol";
+import { L1Block } from "src/L2/L1Block.sol";
+
+/// @title WETH contract that reads the name and symbol from the L1Block contract
```

Comment:
We need to update the L2 genesis generation to put this contract in state instead of the `WETH9` predeploy. That means we have to generate bindings for it. I think leaving the name of this contract to be `WETH` is great.